### ISSUE TITLE
Dropdown menu not working

### ISSUE THREAD

**Issue by gogito** [kasperisager/vanilla-bootstrap#173]
So, I read your answer for this problem here:
http://vanillaforums.org/discussion/27093/how-to-create-custom-dropdown-menu-in-main-nav#latest
And I tried adding this
```
[...]
<div class="navbar-collapse collapse">
  <ul class="nav navbar-nav">
    {home_link}
    {categories_link}
    {discussions_link}
    {activity_link}
    <li class="dropdown">
      <a href="#" role="button" class="dropdown-toggle" data-toggle="dropdown">Dropdown <b class="caret"></b></a>
      <ul class="dropdown-menu" role="menu">
        <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Action</a></li>
        <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Another action</a></li>
        <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Something else here</a></li>
        <li role="presentation" class="divider"></li>
        <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Separated link</a></li>
      </ul>
    </li>
  </ul>
  [...]
</div><!--/.nav-collapse -->
[...]

```
to my master.default.tpl file but it doesn't work as expected, the dropdown doesn't work, it shows the top item but that's it. Am I missing something ?

Hover on it doesn't do anything
![hhj](https://cloud.githubusercontent.com/assets/9802461/8000067/b93861a4-0b7f-11e5-80ad-0e764dd0e0c6.PNG)

**Comment by kasperisager** [comment]
The dropdown JS component is not currently included out of the box as it conflicts with jQuery version used in the latest stable release of Vanilla (2.1.x). This will be fixed when 2.2 comes out.

**Comment by jfarthing84** [comment]
Has this been resolved? Bootstrap 3 requires jQuery 1.9.1 and Vanilla 2.2 appears to be using 1.10.2.
